helm/app/templates/application/app/podmonitor.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.prometheus.podMonitoring -}}
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: {{ .Values.resourcePrefix }}webapp
+  labels:
+    app.service: {{ .Values.resourcePrefix }}webapp
+spec:
+  selector:
+    matchLabels:
+      app.service: {{ .Values.resourcePrefix }}webapp
+  podMetricsEndpoints:
```

Comment:
is this fine to be left empty? (in the event that `metricsEnabled` is false)

Reply:
probably not, I hadn't thought it'd matter as within the person's control

Reply:
I think it will matter in terms of DX